rosa_learning/deploying_application_workshop/cloud-experts-deploying-application-deployment.adoc

```diff
@@ -0,0 +1,175 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="cloud-experts-deploying-application-deployment"]
+= Deploying the OSToy application with Kubernetes
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: cloud-experts-deploying-application-deployment
+
+toc::[]
+
+//rosaworkshop.io content metadata
+//Brought into ROSA product docs 23-JAN-2024
+
+Deploying an application involves creating a container image, storing it in an image repository, and defining Deployment object that uses that image.
+
+Deploying an application involves the following steps:
+* Create the images for the front-end and back-end microservice containers
+* Store the container images in an image repository
+* Create the Kubernetes Deployment object for the application
+* Deploy the application
+
+[NOTE]
+====
+This workshop focuses on application deployment and has users run a remote file which uses an existing image.
+==== 
+
```

Comment:
As I mentioned, for the sake of consistency, you may want to add IDs to each header in the style of `[id="<section-name>_<assembly-id>"]`, so this section would be `[id="retrieving-login-command_cloud-experts-deploying-application-deployment"]`